design/ceph/object/ceph-multisite-overview.md

```diff
@@ -48,6 +48,9 @@ To enable Ceph's multisite, the following steps need to happen.
 3. A master zone in the master zone group needs to be created
 4. An object store needs to be added to the master zone
 
+### Master Zone/Zonegroup
+A multi-site configuration requires a master zone group and a master zone. Additionally, each zone group requires a master zone. Zone groups may have one or more secondary or non-master zones. Even though data is replicated among different RGW servers, the master copy of the realm's metadata such users, quota is stored in the master zone of the master zone group.
```

Comment:
```suggestion
A multi-site configuration requires a master zone group and a master zone. Additionally, each zone group requires a master zone. Zone groups may have one or more secondary or non-master zones. Even though data is replicated among different RGW servers, the master copy of the realm's metadata is stored in the master zone of the master zone group. 
```

Follow this statement up with the conclusion that you would like users to draw from the statement or a side effect of it. It seems to me that's something like "The master zone is assumed to be the source of truth of all data."